### PR TITLE
More unit tests for project management API

### PR DESCRIPTION
**Pull request by hiranya911**
Bumps the test coverage of the `projectmanagement` package from 84% to 92%.

### CODE REVIEW

File: src/test/java/com/google/firebase/projectmanagement/AndroidAppTest.java [hunk 1]
```diff
@@ -189,6 +189,23 @@ public void testDeleteShaCertificateAsync() throws Exception {
...
+    assertEquals(APP_ID, ANDROID_APP_METADATA.getAppId());
+    assertEquals(APP_DISPLAY_NAME, ANDROID_APP_METADATA.getDisplayName());
+    assertEquals(PROJECT_ID, ANDROID_APP_METADATA.getProjectId());
+    assertEquals(APP_PACKAGE_NAME, ANDROID_APP_METADATA.getPackageName());
+  }
+
+  @Test
+  public void testAndroidAppMetadataEquality() {
+    AndroidAppMetadata other =
+        new AndroidAppMetadata(APP_NAME, APP_ID, APP_DISPLAY_NAME, PROJECT_ID, APP_PACKAGE_NAME);
```
Comment (weixifan) [inline src/test/java/com/google/firebase/projectmanagement/AndroidAppTest.java]: nit: Let's stay consistent with the rest of the test and insert a blank line here between the Arrange section and Act + Assert combo section.

File: src/test/java/com/google/firebase/projectmanagement/IosAppTest.java [hunk 2]
```diff
@@ -199,6 +199,27 @@ public void testGetConfigAsyncShouldRethrow() throws Exception {
...
+  }
+
+  @Test
+  public void testIosAppMetadataEquality() {
+    IosAppMetadata other = new IosAppMetadata(
+        TEST_APP_NAME,
+        TEST_APP_ID,
+        TEST_APP_DISPLAY_NAME,
+        TEST_PROJECT_ID,
+        TEST_APP_BUNDLE_ID);
```
Comment (weixifan) [inline src/test/java/com/google/firebase/projectmanagement/IosAppTest.java]: nit: Similarly here.

File: src/test/java/com/google/firebase/projectmanagement/ShaCertificateTest.java [hunk 3]
```diff
@@ -58,4 +58,21 @@ public void getTypeFromHashSha256WithIncorrectSize() {
     ShaCertificate.getTypeFromHash(
         "1111AAAA1111AAAA1111AAAA1111AAAA1111AAAA1111AAAA1111AAAA");
   }
+
+  @Test
+  public void testEquality() {
+    ShaCertificate shaOne = ShaCertificate.create("1111AAAA1111AAAA1111AAAA1111AAAA1111AAAA");
+    ShaCertificate shaTwo = ShaCertificate.create("1111AAAA1111AAAA1111AAAA1111AAAA1111AAAA");
```
Comment (weixifan) [inline src/test/java/com/google/firebase/projectmanagement/ShaCertificateTest.java]: nit: Similarly here.

File: src/test/java/com/google/firebase/projectmanagement/ShaCertificateTest.java [hunk 4]
```diff
@@ -58,4 +58,21 @@ public void getTypeFromHashSha256WithIncorrectSize() {
...
+    assertEquals(shaOne, shaTwo);
+    assertEquals(shaOne.hashCode(), shaTwo.hashCode());
+  }
+
+  @Test
+  public void testToString() {
+    ShaCertificate sha = ShaCertificate.create("cert", "1111AAAA1111AAAA1111AAAA1111AAAA1111AAAA");
```
Comment (weixifan) [inline src/test/java/com/google/firebase/projectmanagement/ShaCertificateTest.java]: nit: Similarly here.

Comment (hiranya911) [reply src/test/java/com/google/firebase/projectmanagement/ShaCertificateTest.java]: Done

File: src/test/java/com/google/firebase/projectmanagement/ShaCertificateTest.java (hunk 3 above)
Comment (hiranya911) [reply src/test/java/com/google/firebase/projectmanagement/ShaCertificateTest.java]: Done

File: src/test/java/com/google/firebase/projectmanagement/IosAppTest.java (hunk 2 above)
Comment (hiranya911) [reply src/test/java/com/google/firebase/projectmanagement/IosAppTest.java]: Done

File: src/test/java/com/google/firebase/projectmanagement/AndroidAppTest.java (hunk 1 above)
Comment (hiranya911) [reply src/test/java/com/google/firebase/projectmanagement/AndroidAppTest.java]: Done

**Pull request merged**